GHAS-on-GHES-feature-matrix.md

```diff
@@ -102,76 +106,78 @@ Dependabot alerts tell you that your code depends on a package that is insecure.
-|[Security Overview](https://docs.github.com/en/enterprise-server/code-security/security-overview/about-security-overview)|✅|✅|✅|✅|✅|✅|✅|✅|
-|Organization view|☑️|✅|✅|✅|✅|✅|✅|✅|
+|Feature  |3.4 |3.5 |3.6 |3.7 |3.8 |3.9 |3.10| 3.11 |3.12|
+|------------------------------------------------------------|-----|-----|-----|-----|-----|-----|----|----|----|
+|[Security Overview](https://docs.github.com/en/enterprise-server/code-security/security-overview/about-security-overview)|✅|✅|✅|✅|✅|✅|✅|✅|✅|
+|Organization view|☑️|✅|✅|✅|✅|✅|✅|✅|✅|
 |Enterprise view||☑️|☑️|✅|✅|✅|✅|✅|
-|Organization-level Code Scanning Alert View||✅|✅|✅|✅|✅|✅|✅|
-|Organization-level Dependabot Alert View||✅|✅|✅|✅|✅|✅|✅|
-|Enterprse-level view of Dependabot alerts|||✅|✅|✅|✅|✅|✅|
-|Enterprse-level view of code scanning alerts||||✅|✅|✅|✅|✅|
-|Enterprse-level view of secret scanning alerts||||✅|✅|✅|✅|✅|
-|Coverage and Risk Security Overview pages|||||☑️|☑️|✅|✅|
-|[Filter alerts by repo topic](https://docs.github.com/en/enterprise-server/code-security/security-overview/filtering-alerts-in-security-overview)||||||✅|✅|✅|
-|[Filter alerts by team](https://docs.github.com/en/enterprise-server/code-security/security-overview/filtering-alerts-in-security-overview)||||||✅|✅|✅|
-|[Enable GHAS features in security overview](https://docs.github.com/en/enterprise-server/code-security/security-overview/about-security-overview)||||||✅|✅|✅|
-|[Enterprise-level security coverage and risk dashboards](https://docs.github.com/en/enterprise-server@3.10/code-security/security-overview/about-security-overview#about-security-overview-for-enterprises)|||||||✅|✅|
+|Organization-level Code Scanning Alert View||✅|✅|✅|✅|✅|✅|✅|✅|
+|Organization-level Dependabot Alert View||✅|✅|✅|✅|✅|✅|✅|✅|
+|Enterprse-level view of Dependabot alerts|||✅|✅|✅|✅|✅|✅|✅|
+|Enterprse-level view of code scanning alerts||||✅|✅|✅|✅|✅|✅|
+|Enterprse-level view of secret scanning alerts||||✅|✅|✅|✅|✅|✅|
+|Coverage and Risk Security Overview pages|||||☑️|☑️|✅|✅|✅|
+|[Filter alerts by repo topic](https://docs.github.com/en/enterprise-server/code-security/security-overview/filtering-alerts-in-security-overview)||||||✅|✅|✅|✅|
+|[Filter alerts by team](https://docs.github.com/en/enterprise-server/code-security/security-overview/filtering-alerts-in-security-overview)||||||✅|✅|✅|✅|
+|[Enable GHAS features in security overview](https://docs.github.com/en/enterprise-server/code-security/security-overview/about-security-overview)||||||✅|✅|✅|✅|
+|[Enterprise-level security coverage and risk dashboards](https://docs.github.com/en/enterprise-server@3.10/code-security/security-overview/about-security-overview#about-security-overview-for-enterprises)|||||||✅|✅|✅|
 
 ## Administration
-|Feature  |3.4 |3.5 |3.6 |3.7 |3.8 |3.9 |3.10 |3.11 |
+|Feature  |3.4 |3.5 |3.6 |3.7 |3.8 |3.9 |3.10 |3.11 |3.12 |
 |------------------------------------------------------------|-----|-----|-----|-----|-----|-----|----|----|
```

Comment:
```suggestion
|------------------------------------------------------------|-----|-----|-----|-----|-----|-----|----|----|----|
```

Reply:
breaks MD table render without this header:

<img width="565" alt="image" src="https://github.com/advanced-security/advanced-security-material/assets/1760475/aa91b456-e53c-41a6-b98c-39448181c4aa">